### PR TITLE
Feature/iii 3566 status button 2

### PR DESCRIPTION
**Pull request by simon-debruijn**
### Added
- status-button place-detail
- status-button event-detail
- translations

---
Ticket: https://jira.uitdatabank.be/browse/III-3566

Screenshot:
Schermafbeelding 2021-03-03 om 15 47 41

### CODE REVIEW

File: src/place-detail/ui/place-detail.html [hunk 1]
```diff
@@ -29,6 +29,9 @@ <h1 class="title" ng-bind="::place.name"></h1>
                 class="list-group-item"
                 type="button"
                 ng-click="openTranslatePage()"><i class="fa fa-globe" aria-hidden="true"></i>  <span translate-once="preview.translate"></span></button>
+        <a ng-if="::permissions.editing"
+           class="list-group-item"
+           ng-href="{{ place.url + '/status' }}"><i class="far fa-calendar-check" aria-hidden="true"></i>  <span translate-once="preview.change_status"></span></a>
```
Comment (brampauwelyn) [inline src/place-detail/ui/place-detail.html:34]: Is it normal the class is `far` ? Or is it a typo?

**Review by brampauwelyn — APPROVED** [review]

File: src/place-detail/ui/place-detail.html (hunk 1 above)
Comment (bertramakers) [reply src/place-detail/ui/place-detail.html:34]: I think that's the class name for the regular icon set. https://fontawesome.com/how-to-use/on-the-web/referencing-icons/basic-use

**Review by bertramakers — CHANGES_REQUESTED** [review]
@simon-debruijn can you add an appconfig parameter for this, so we can hide it on production after we merge & deploy it? Since we might need to deploy Angular changes in the meantime even if the offer status pages are not 100% tested and done. For example for hotfixes etc.

**Review by bertramakers — APPROVED** [review]
I was overruled by Corneel, so go ahead 😄

**Pull request merged**